app/build.gradle

```diff
@@ -21,5 +21,9 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:support-v4:25.3.1'
+    compile "com.android.support:support-v4:$rootProject.supportLibraryVersion"
+
+    // UI Related
+    compile "com.jakewharton:butterknife:$rootProject.butterKnifeVersion"
+    annotationProcessor "com.jakewharton:butterknife-compiler:$rootProject.butterKnifeVersion"
```

Comment:
I bet this line is the one conflicting with master, please rebase and fix it :wink: